<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hardAESwidget</class>
 <widget class="QWidget" name="hardAESwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>文件加密保护</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>是否需要生成密钥？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="genpushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>生成</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filelineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="filepushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>硬件AES文件加解密</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="keylineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="keypushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择密钥</string>
   </property>
  </widget>
  <widget class="QPushButton" name="encryptpushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>加密</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decryptpushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>解密</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>421</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="backpushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>550</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
